src/app/helptext/system/ssh-connections.ts

```diff
@@ -0,0 +1,38 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    setup_method_placeholder: T('Setup Method'),
+    setup_method_tooltip: T(''),
+
+    host_placeholder: T('Host'),
+    host_tooltip: T(''),
+
+    port_placeholder: T('Port'),
+    port_tooltip: T(''),
+
+    username_placeholder: T('Username'),
+    username_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
+
+    remote_host_key_placeholder: T('Remote Host Key'),
+    remote_host_key_tooltip: T(''),
```

Comment:
Connection key generated from the remote system used to authenticate with this system. Copy the contents of the remote system private key and paste the contents in this field.

Reply:
No, thats the host key, its the key of the remote ssh server. There should be a button to scan the host key of the remote node.